src/components/auth/auth.tsx

```diff
@@ -0,0 +1,33 @@
+import { useAuth } from '@/hooks/useAuth';
+
+import { EAppPage } from '../../constants/link-urls';
+import { AppLink } from '../app-link';
+import { EIconType, Icon } from '../icons';
+
+import styles from './auth.module.css';
+
+const BUTTON_CLASSNAME = styles['auth-buttons__button'];
+
+export const Auth = () => {
+    const { isAuthorized, onLogout } = useAuth();
+
+    return (
+        <div className={styles['auth-buttons']}>
+            {isAuthorized ? (
+                <button onClick={onLogout} className={`${BUTTON_CLASSNAME} ${styles['auth-buttons__button_logout']}`}>
```

Comment:
- ... and here could be simple 

```suggestion
                <button onClick={onLogout} className={`${styles.button} ${styles.logout}`}>
```
or use clsx